src/main/java/com/google/sps/servlets/GoogleMapkeyServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** 
+ * Servlet sends Google maps API key as a String.
+ * */
+@WebServlet("/google-map-key")
+public class GoogleMapkeyServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println(System.getenv("GOOGLE_MAPS_API_KEY"));
+    }
+}
```

Comment:
We could place the api key along with the event list (at `EventRetrievalServlet.java`) but it would be sent every time there is a search and we only need it every time index.html loads.